rlbot_gui/gui.py

```diff
@@ -225,6 +227,46 @@ def download_bot_pack():
     settings.setValue(BOT_FOLDER_SETTINGS_KEY, bot_folder_settings)
     settings.sync()
 
+@eel.expose
+def download_bot(repo):
+
+    branch = ""
+    if "tree" in repo:
+        branch = repo.split('/')[-1]
+        repo = repo.split('/tree')[0]
+    else:
+        branch = "master"
+
+    download_botfs(
+        repo_url=repo,
+        checkout_folder='download',
+        branch_name=branch,
+        bot_path=os.path.abspath(BOTPACK_FOLDER))
+    return 0;
+
+@eel.expose
+def delete_bot(name):
+    shutil.rmtree(os.path.abspath(BOTPACK_FOLDER + "/" + name))
+    return 0;
+
+@eel.expose
+def isBotInstalled(bot_name):
```

Comment:
is this required to be camel case?

Reply:
no, its not. can be renamed ofc, just make bure it also gets edited on botmanager.js

Reply:
solved